Scripts/RunningCharacter.cs

```diff
@@ -0,0 +1,64 @@
+using System;
+using Godot;
+
+public class RunningCharacter : Spatial
```

Comment:
Can rename as `RunningCharacterAnimation`? Or something that indicates it's for animation?

Reply:
Its technically the mesh + animation. Can call it RunningSprite maybe?

Reply:
Or MainCharacterSprite?

Reply:
Isn't sprite a 2d image

Reply:
or `MainCharacterModel` ah

Reply:
a computer graphic which may be moved on-screen and otherwise manipulated as a single entity.

Reply:
anything bah

Reply:
ok you just decide whichever better